AotCompatibility/AotCompatibility.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+	<InvariantGlobalization>true</InvariantGlobalization>
+	  
+    <PublishAot>true</PublishAot>
```

Comment:
Add:
```
		<IsTrimmable>true</IsTrimmable>
```

Reply:
Doesn't `PublishAot` cover that?

Reply:
https://learn.microsoft.com/en-nz/dotnet/core/deploying/trimming/prepare-libraries-for-trimming?pivots=dotnet-8-0#enable-project-specific-trimming

> The IsTrimmable property defaults to true when configuring a project as AOT-compatible with `<IsAotCompatible>true</IsAotCompatible>`.

Do we need to set that separately from `PublishAot`?

Reply:
I'm not sure. @sergio0694 always tells me to include it, perhaps he can give better guidance.

Reply:
If you use `PublishAot`, that's sufficient. `IsAotCompatible` is used for libraries, but if you're making an app, the former covers everything you need already 🙂